docs/content/docs/configuration.md

```diff
@@ -51,6 +54,118 @@ status:
 default_decision: /http/example/authz/allow
 ```
 
+## Environment Variable Substitution
+> Only supported with the OPA runtime (`opa run`).
+
+Environment variables referenced with the `${...}` notation within the configuration
+will be replaced with the value of the environment variable.
+
+Example using `BASE_URL` and `BEARER_TOKEN` environment variables:
+```yaml
+services:
+  - name: acmecorp
```

Comment:
Nit: Let's just use the map syntax going forward to avoid issues with overriding values inside arrays.

Reply:
+1